payment/migrations/0021_referralcoupon.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.5 on 2022-09-16 03:41
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('payment', '0020_alter_coupon_options_coupon_polymorphic_ctype'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ReferralCoupon',
```

Comment:
Nice work architecting this and the polymorphic coupon model; at some point will want to document the architecture of Learn so new engineers will understand how things work. But maybe not high-pri for now since team not growing so fast yet.